aict_tools/io.py

```diff
@@ -194,14 +283,41 @@ def __next__(self):
         start = chunk * self.chunksize
         end = min(self.n_rows, (chunk + 1) * self.chunksize)
         self._current_chunk += 1
+        if self.current_key > 0:
```

Comment:
Isn't this more elegantly done with just remembering how many event's were already read?

Reply:
But that doesn't give you the table name, or?

Reply:
You could just iterate over an iterator of the tables and get the next one when necessary: